kitchen-inspec.gemspec

```diff
@@ -20,11 +20,12 @@ Gem::Specification.new do |spec|
   spec.bindir        = "exe"
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
+  spec.required_ruby_version = ">= 2.1.0"
   spec.add_dependency "inspec", ">=0.22.0", "<1.0.0"
   spec.add_dependency "test-kitchen", "~> 1.6"
   spec.add_development_dependency "countloc", "~> 0.4"
   spec.add_development_dependency "bundler", "~> 1.10"
-  spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_development_dependency "rake", "~> 11.0"
```

Comment:
I think we should remove the development dependencies from gemspec and place them into gemfile. @tas50 What do you think?

I think we should only have:
```
  spec.add_dependency "inspec", ">=0.22.0", "<1.0.0"
  spec.add_dependency "test-kitchen", "~> 1.6"
```
here